docker-compose.yml

```diff
@@ -21,8 +22,9 @@ services:
       - '3000:3000'
     volumes:
```

Comment:
Because the file system in container is independent from the host system, so once the container is deleted, all files are gone. And the 'stateless' is the key spirit of containerize design (which means you can just delete the container and update with the new container at any time.)

But we still need to store persistent data, so that's what volumes stand for. Volume can mount a folder from the host to the container, so if the container is deleted, the mounted folder will persist. In our cases, I use volumes to mount source code into container, so we can edit source code and the program will updates right away.

You can learn more about volume by docker's guide: https://docs.docker.com/storage/